docs/src/components/Layout/FrameworkChooser.tsx

```diff
@@ -46,6 +46,7 @@ export const FrameworkChooser = ({ platform }) => {
       <ToggleButton
         value="react"
         size="small"
+        title="React"
         padding={{ base: '4px', medium: undefined }}
       >
         <VisuallyHidden>React</VisuallyHidden>
```

Comment:
Thanks for adding the tooltips!

Do we want to reduce the repeated words or even combine the `ToggleButton`s? It might be out of the scope of this PR, but we might not have a ticket to do it later